composer.json

```diff
@@ -27,6 +27,7 @@
         "drupal/entity_reference_revisions": "^1.9",
         "drupal/extra_field": "^2.1",
         "drupal/facets_form": "1.0.0-alpha3",
+        "drupal/field_display_label": "^1.3",
```

Comment:
Yes, I didn't ask yet because if we get rid of the label differences, we can drop the module altogether.
This module doesn't have dedicated tests either.